pkg/jmxfetch/jmxfetch_windows.go

```diff
@@ -8,3 +8,8 @@
 package jmxfetch
 
 func (j *JMXFetch) Monitor() {}
+
+// Stop stops the JMXFetch process
+func (j *JMXFetch) Stop() error {
+	return j.cmd.Process.Kill()
```

Comment:
you don't want to implement the `Wait()` like we do with unices? If I read the sources correctly (maybe I didn't) the `Kill()` is not guaranteed.

Reply:
Makes sense. Added that!